packages/gatsby-cli/src/reporter/prepare-stack-trace.ts

```diff
@@ -13,11 +13,14 @@ import {
 } from "source-map"
 
 export class ErrorWithCodeFrame extends Error {
-  codeFrame: string
+  codeFrame = ``
 
-  constructor(...args) {
-    super(...args)
-    this.codeFrame = ``
+  constructor(error) {
```

Comment:
Not sure, but we are missing type info here?

```suggestion
  constructor(error: Error) {
```

Shouldn't this cause eslint problems? Does `// @ts-ignore` used in this file ignoring everything and not just the nearest lines?

Reply:
I don't think we have lint rules on function arguments, we only have it on return types being required.. But yeah this is a good idea